spec/models/druid_version_zip_spec.rb

```diff
@@ -25,6 +24,50 @@
     end
   end
 
+  describe '.create_zip!' do
```

Comment:
it seems we should have tests somewhere that show the directory structure inside the zip is what we want.

And possibly to show that it writes it to the right directory?  (-p option added)

Reply:
The specs are mainly concerned with the making the zip command what we expect.  They do not intend to test `zip` itself deeply.  While possible, testing one version of `zip` is meager reassurance, given that we have a system anticipating using multiple different versions of `zip` over time (and simultaneously!).  I'll put some work in this direction, but given the flexibility of the interface w/ the system utils, I don't expect to ever make this bulletproof.

Line 38 already does test that the zip is created in the right place, which implies that the necessary directory was created (else it would raise, like it did when we tried to use it in stage).

Reply:
@ndushay To do this correctly seems like it would require including a gem like `zip`... in test only, used only for one test.  Does that seem worthwhile?

Reply:
I guess what struck me is that the internal directory structure is really importan, so having a test to show it might be a good thing, long term maint-wise.  What if we just shell out to unpack it and then read the unpacked dir with ruby to check the structure is what we expect?

If we start having multiple zip commands, then the zipmaker code will have to accommodate ... so new code then implies additional tests then as well, no?

Reply:
Yeah, this is what I mentioned to you at close last night, I think I have a workaround (using `unzip -l ...` process return value like `ls`).  Pursuing that. 

We don't intend to have multiple zip **commands**, we expect that over time, the `zip` utility itself will be updated (or differ on how it was compiled on a given OS).

Reply:
added test of internal structure